src/components/button.tsx

```diff
@@ -16,6 +16,7 @@ export interface ButtonProps {
     icon: IconProp;
     fontSize: number;
     onClick(e: MouseEvent<HTMLElement>): void;
+    disabled: boolean;
```

Comment:
Can we rename this to something like `defaultDisabled`? Just to distinguish this from the state's disabled.